docs/auroLabs.md

```diff
@@ -0,0 +1,62 @@
+<style>
+  .lightText {
+    color: var(--auro-color-text-secondary-on-light);
+    margin-bottom: 2rem;
+  }
+
+  .pre {
+    color: var(--auro-color-brand-flamingo-500);
+    font-size: 0.8rem;
+    font-family: monaco;
+    padding-top: .5rem;
+    margin-bottom: 0;
+  }
+
+  .trigger {
+    font-size: 1.2rem;
+  }
+</style>
+
+# AuroLabs: BETA
+
+Building a design system is dependent on the contribution of others. From the very beginning Auro positioned itself to be open and welcome to contributions by making it open source.
+
+But making a project open source isn't enough when it comes to growing a community based tool. Welcome to AuroLabs!
+
+## Please don't mind the mess
+
+We are rolling this out as a BETA to iron out the kinks. Your patience in this is welcome. As we go through this process we will add more definition to the process to ensure transparency versus creating a black box.
```

Comment:
```suggestion
We are rolling this out as a BETA to iron out the kinks. Your patience in this is welcome. As we go through this process we will add more definition to the process to ensure transparency instead of creating a black box.
```